applications/feedback_sentiment_analyzer/README.md

```diff
@@ -0,0 +1,42 @@
+# Feedback Sentiment Analyzer (FSA)
+
+Feedback Sentiment Analyzer (FSA) is an example application that showcases AWS services and SDKs.
+
+Specifically, it solves a ficticious use case of a hotel in Paris, France, which receives feedback from guests in a variety of foreign languages.
+
+## SDKs this application is implemented in
+The underlying business logic has been implemented in the following AWS SDKs:
+* [Ruby](../../ruby/cross-services/feedback-sentiment-analyzer/README.md)
+
+
+## AWS services used
+
+This application uses a suite of [machine learning on AWS](https://aws.amazon.com/machine-learning/) to:
+* extract text using [Amazon Textract](https://aws.amazon.com/textract/).
+* detect sentiment using [Amazon Comprehend](https://aws.amazon.com/comprehend/).
+* translate to French using [Amazon Translate](https://aws.amazon.com/translate/).
+* synthesize to human-like speech using [Amazon Polly](https://aws.amazon.com/polly/).
+
+Additionally, the application showcases:
+* [Amazon S3]() to store images of comment cards
+* [AWS Lambda]() to execute business logic
+* [Step Functions]() to orchestrate multiple Lambda functions 
+* [Amazon DynamoDB]() to store details about each comment
+
+## Deployment instructions
+This application is deployed using the [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/).
+
+1. Navigate to [/cdk](/client)
+1. Run `npm install -i`
+1. Export the following variables:
+    ```bash
```

Comment:
Add other operating systems like we did with PAM

Reply:
I don't see any example of this in the PAM readme. Can you elaborate?

Reply:
It's in the cdk readme.
_bash/zsh_

```
export PAM_NAME=
export PAM_EMAIL=
export PAM_LANG=
```

_Windows cmd_

```
set PAM_NAME=
set PAM_EMAIL=
set PAM_LANG=
```

_Windows Powershell_

```
$Env:PAM_NAME =
$Env:PAM_EMAIL =
$Env:PAM_LANG =
```

Reply:
Done